### DISCUSSION TITLE
All agents show as overdue

### DISCUSSION THREAD

**Discussion by ebdavison** [amidaware/tacticalrmm#1233] [Q&A] [answered]
**Server Info (please complete the following information):**
 - OS: Ubuntu 20.04 LTS
 - Browser: chrome
 - RMM Version (as shown in top left of web UI): 0.14.3

**Installation Method:**
  - [X] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): 0.1.0
- Agent OS: 2x Windows 10; 1x Windows Server 2012 R2

**Describe the bug**
After a brand new install, added 2 Windows 10 and 1 Windows Server 2012 R2 agents and generally they seem to be working as the checks continue to come in and the software list works and I can do a remote session, etc.  

However, on the main screen the system shows as overdue and last boot time 53 years ago.

Also, none of the hardware details are showing up with a note that a newer agent version is needed for the graphics info.

**To Reproduce**
1. Go to main screen.
2. Check Last Response and Boot Time info.

**Expected behavior**
Agents, after install, will continue to check in properly and all information that should be populated or updated, does so properly.

**Screenshots**
![image](https://user-images.githubusercontent.com/549431/181937558-940770a8-ce77-4ab9-a9ca-b3596a1a7f5c.png)

![image](https://user-images.githubusercontent.com/549431/181937624-53f2e9a3-95fa-4255-9a55-1539e84450da.png)

**Additional context**
Unknown

**Comment by wh1te909** [comment, marked as answer]
Please read through the troubleshooting docs and provide the steps you've taken to troubleshoot, describe your environment in detail (where is your VM running? what kind of firewalls etc) provide some logs.

Your issue is related to the `nats.service` and `nats-api.service` specifically (check the How it All Works page in docs to troubleshoot these) and/or websockets (check nginx logs)

**Comment by ebdavison** [comment]
Sure enough.  The `nats` service was not running.  There had been a problem with the SSL certs when building the box and I had to renew them manually.  When I did, the owner of the files had changed from `tactical` to `root` and nats could not read the certificates with `permission denied`.  I fixed the permissions and got the `nats` service running and now agents are checking in and showing online with their details coming in as well.

Thank you for the pointer on where to look!

Nice product so far and I hope to explore whether it will fit my needs now that it is connecting properly.

**Comment by ebdavison** [comment]
![image](https://user-images.githubusercontent.com/549431/181996452-4eed8e69-3c7b-41ce-acca-64ae04b58230.png)

![image](https://user-images.githubusercontent.com/549431/181996465-a0193676-abc9-4dcb-aed0-0194ec3c08d7.png)

Although the boot time still shows 53 years ...

**Reply by dinger1986** [reply]
It might need the agent reinstalled or it's not gathered all the information yet.

Can you send commands to it?

**Reply by ebdavison** [reply]
Yes, I can send commands, get data back, check work and I can see what looks like 5-minute interval data (disk space,etc.) as well as remote control.

**Reply by ebdavison** [reply]
Looks like I just needed to be patient.  Right after my response I checked the console again and they look to be up to date now.

![image](https://user-images.githubusercontent.com/549431/182003985-8cbf8d41-7844-4e48-b80e-de9c4596a6f2.png)

**Reply by dinger1986** [reply]
Ok perfect.

Glad it's working properly. Enjoy 😃
